<launch>
    <arg name="latency" default="0.5"/>
    <arg name="close_distance_factor" default="0.8"/>
    <arg name="medium_distance_factor" default="0.5"/>
    <arg name="far_distance_factor" default="0.3"/>

    <node name="calibration_controller" pkg="robot_assisted_calibration" type="CalibrationController.py" output="screen">
        <param name="latency" type="double" value="$(arg latency)"/>
        <param name="close_distance_factor" type="double" value="$(arg close_distance_factor)"/>
        <param name="medium_distance_factor" type="double" value="$(arg medium_distance_factor)"/>
        <param name="far_distance_factor" type="double" value="$(arg far_distance_factor)"/>
    </node>
</launch>